pkg/constants/constants.go

```diff
@@ -219,6 +220,11 @@ func DefaultServiceName(name string, component InferenceServiceComponent) string
 	return name + "-" + component.String() + "-" + InferenceServiceDefault
 }
 
+//TODO: need to add shard id to the configmap name
+func DefaultMultiModelConfigMapName(inferenceserviceName string, shardId int) string {
```

Comment:
What does Default mean in this context? Can it be overridden? Is this juxtaposed against canary?

Reply:
You could also define this as a format string instead of a function which is a little more "constant friendly"

const MultiModelConfigMapNameFormat = "multimodelconfig-%s-%d"

Name = fmt.Sprint(constants.MultiModelConfigMapNameFormat , isvc.Metadata.name, shard)

Reply:
Actually, does MMS need to differentiate between default and canary? User can just deploy a different TrainedModel to try it out. Let me rename it to `MultiModelConfigMapName`

Reply:
I'm asking you. IIUC "Default" is redundant, unless I'm misunderstanding. Canary isn't in the v1beta1 spec.

Reply:
nit: if its a func, should be starting with a "verb" ?